libcontainer/nsenter/nsexec.c

```diff
@@ -89,14 +89,21 @@ struct nlconfig_t {
 	size_t gidmappath_len;
 };
 
-#define PANIC   "panic"
-#define FATAL   "fatal"
-#define ERROR   "error"
-#define WARNING "warning"
-#define INFO    "info"
-#define DEBUG   "debug"
+/*
+ * Log levels are the same as in logrus.
+ */
+#define PANIC   0
+#define FATAL   1
+#define ERROR   2
+#define WARNING 3
+#define INFO    4
+#define DEBUG   5
+#define TRACE   6
+
+static const char *level_str[] = { "panic", "fatal", "error", "warning", "info", "debug", "trace" };
 
 static int logfd = -1;
+static int loglevel = INFO;
```

Comment:
This should default to `DEBUG` -- if there's hypothetically an error very early on in parsing, we wouldn't be able to get debug logs even if we passed `--debug` to `runc`.

Reply:
Makes sense. Done.